internal/provider/common/runtimesource.go

```diff
@@ -19,6 +20,7 @@ type NativeRuntimeModel struct {
 
 type DockerRuntimeSourceModel struct {
 	AutoDeploy           types.Bool        `tfsdk:"auto_deploy"`
+	AutoDeployTrigger types.String `tfsdk:"auto_deploy_trigger"`
```

Comment:
nit: seems like gofmt didn't run here. Looks like the linter workflows were not running on this PR. I had to click an "approve and run workflows button", and now the linter is running and failing.

Reply:
I think it should be... I'm not entirely sure what happened.